### PR TITLE
Popup branch

### PR DESCRIPTION
**Pull request by mhamedGd**
- Added 'Dynamically Created Project Cards' in Javascript.
- Added Popup Details for each card for both mobile and desktop.
- Added some interactions with project cards.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,6 +32,31 @@
...
+                    <button class="link-styler">
+                      See Live                      
+                    </button>
+                    <button class="link-styler">See Source</button>
+                  </div>
+                </div>
+              </div>
+              <button class="project-details-popup-window-cancel"><img src="./rsc/icons/Icon-Cancel.svg" alt="Exit Popup Window"></button>
+            </div>
+          </div>
```
Comment (Qoosim) [inline index.html:59]: - [ ] All the already created HTML code for projects list should be removed, and instead it should be added via JavaScript
- [ ] The popup should be added to the first project too. Presently, there is no popup effect the the see project button is clicked. Kindly attend to it. Please take a look at what I am saying.


https://user-images.githubusercontent.com/26597432/202405674-6101806f-a50b-411f-9749-70763d0c828c.mp4

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @mhamedGd, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Kindly move the project name a bit down on your popup modal on your mobile version of the project. Please, take a look at what I am saying.

mobile-popup-issue-1

File: index.html (hunk 1 above)
Comment (mhamedGd) [reply index.html:59]: Alright, thank you. for the first point, What you highlighted is the popup window, not the project cards list. The project cards are all removed and you can see the function in javascript.
For the second point, I will add it to the main project button to be able to see the details.

Comment (leonardodiasb) [inline index.html:59]: - [ ] As the previous code reviewer mentioned all the already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript. You mentioned that this code is the popup window code, but the popup window should contain the information related to the project the user clicks on, not a hardcoded HTML data. The user should be able to click on the project and see more information in the popup like the links to the GitHub or a live version, so creating it dynamically with Javascript would be a better idea, and just toggling it on and off it is not a good approach. Kindly fix that.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @mhamedGd, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct GitHub Flow. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mhamedGd) [reply index.html:59]: Great! Thank you, done.

**Pull request merged**